spec/lib/extensions/ar_region_spec.rb

```diff
@@ -127,4 +127,29 @@
       expect(ManageIQ::Providers::Vmware::InfraManager::Vm.new.split_id).to eq([ManageIQ::Providers::Vmware::InfraManager::Vm.my_region_number, nil])
     end
   end
+
+  describe "#discover_my_region_number" do
```

Comment:
Technically you are testing the method `.my_region_number`, so you should probably describe that and not the private method.